components/activity_log_modal/components/more_info.tsx

```diff
@@ -22,7 +22,7 @@ type Props = {
     moreInfo: boolean;
 };
 
-export default function MoreInfo(props: Props) {
+export default (props: Props) => {
```

Comment:
This should be given a name somehow since it will otherwise be unnamed in dev tools and [snapshots](https://github.com/mattermost/mattermost-webapp/pull/5901/files#diff-917a01dc9366f5c92a276e4d5572ab28R47). Maybe something like
```
const MoreInfo = () => {...};
export default MoreInfo;
```
would work?

Reply:
👍 I'll change it to this. Was because of function naming convention. Should we change `naming-convention` to allow PascalCase for functions?

Reply:
Yeah, that'd be fine. If there was any way to only allow it for top level functions, that'd be nice, but I assume it's not that specific.